packages/contracts/docs/osx/02-how-to-guides/02-plugin-development/04-upgradeable-plugin/04-subsequent-builds.md

```diff
@@ -0,0 +1,141 @@
+---
+title: Creating Subsequent Builds for Upgradeable Plugins
+---
+
+## How to create a subsequent build to an Upgradeable Plugin
+
+A build is a new implementation of your Upgradeable Plugin. Upgradeable contracts offer advantages because you can cheaply change or fix the logic of your contract without losing the storage of your contract.
+
+In this how to guide, we'll go through how we can create these builds for our plugins. Specifically, we'll showcase two specific types of builds - one that modifies the storage of the plugins, another one which modifies its bytecode. Both are possible and can be implemented within the same build implementation as well.
+
+### 1. Make sure your previous build is deployed and published
+
+Make sure you have at least one build already deployed and published into the Aragon protocol. Make sure to check out our [publishing guide](../07-publication/index.md) to ensure this step is done.
+
+### 2. Create a build implementation modifying storage
+
+In this second build, we want to change certain sections of our existing plugin implementation - we want to update the storage of our plugin with new values. In this specific case, we will add a second storage variable `address public account;`. Additional to the `initializeFromBuild2` function, we also want to add a second setter function `storeAccount` that uses the same permission as `storeNumber`.
```

Comment:
As above, our examples no longer have the `initializeFromBuild2`. Instead we have `initialize(uint8[3] fromVersion, ...)` but only after the first build of a release

Reply:
See:
https://github.com/aragon/simple-storage-example-plugin/blob/main/contracts/release1/build3/SimpleStorageR1B3.sol#L30-L50

Reply:
Simple storage plugin update - added to notes!